.circleci/config.yml

```diff
@@ -189,7 +189,6 @@ workflows:
       - hmpps/deploy_env:
           name: deploy_dev
           env: 'dev'
-          jira_update: true
```

Comment:
Is it worth squashing this into the last commit since you're effectively setting then unsetting these `jira_` values? You did remove `helm_timeout` in the last commit, for instance, even though all the rest of the commented out values were directly uncommented out

Reply:
I've squashed into one commit and updated the commit message to hopefully provide some helpful context.